Source/WebCore/html/canvas/WebGLRenderingContextBase.cpp

```diff
@@ -1165,8 +1166,8 @@ void WebGLRenderingContextBase::bufferData(GCGLenum target, std::optional<Buffer
     if (!buffer)
         return;
 
-    std::visit([&](auto& data) {
-        m_context->bufferData(target, data->span(), usage);
+    std::visit([context = protectedGraphicsContextGL(), target, usage](auto& data) {
```

Comment:
`context = m_context`. We don't normally use `protectedFoo()` functions to initialize variables.